File: 240306/007/007.py
class Mission:
  def __init__(self, code, location, time):
    self.code = code
    self.location = location
    self.time = time

secret_code, meeting_location, appointment_time = tuple(input().split())
kwargs = (secret_code, meeting_location, int(appointment_time))

mission = Mission(*kwargs)
print('secret code :', mission.code)
print('meeting point :', mission.location)
print('time :', mission.time)

File: 240422/007/007.js
const fs = require('fs')
const input = fs.readFileSync(0).toString().trim().split(' ')

class Mission {
    constructor(code,place,time) {
        this.code = code;
        this.place = place;
        this.time = time;
    }
}

const mission1 = new Mission(input[0], input[1], Number(input[2]))

console.log(`secret code : ${mission1.code}`)
console.log(`meeting point : ${mission1.place}`)
console.log(`time : ${mission1.time}`)